src/components/Documents.tsx

```diff
@@ -16,43 +43,42 @@ function Documents() {
       background="#FFFFFF"
     >
       <GridItem>
-        <Image src="/blue.svg" alt="noimage" />
-      </GridItem>
-      <GridItem>
-        <Image src="/grey_rectangle.svg" alt="noimage" />
-      </GridItem>
-      <GridItem>
-        <Image src="/grey_rectangle.svg" alt="noimage" />
-      </GridItem>
-      <GridItem>
-        <Image src="/grey_rectangle.svg" alt="noimage" />
-      </GridItem>
-      <GridItem>
-        <Image src="/grey_rectangle.svg" alt="noimage" />
-      </GridItem>
-      <GridItem>
-        <Image src="/grey_rectangle.svg" alt="noimage" />
-      </GridItem>
-      <GridItem>
-        <Image src="/grey_rectangle.svg" alt="noimage" />
-      </GridItem>
-      <GridItem>
-        <Image src="/grey_rectangle.svg" alt="noimage" />
-      </GridItem>
-      <GridItem>
-        <Image src="/grey_rectangle.svg" alt="noimage" />
-      </GridItem>
-      <GridItem>
-        <Image src="/grey_rectangle.svg" alt="noimage" />
-      </GridItem>
-      <GridItem>
-        <Image src="/grey_rectangle.svg" alt="noimage" />
-      </GridItem>
-      <GridItem>
-        <Image src="/grey_rectangle.svg" alt="noimage" />
+        <Image src="/blue.svg" alt="noimage" onClick={onOpen} />
+        <Modal isOpen={isOpen} onClose={onClose}>
+          <ModalOverlay />
+          <ModalContent>
+            <ModalHeader>New Document</ModalHeader>
+            <ModalCloseButton />
+            <ModalBody>
+              {" "}
```

Comment:
This is a nitpick, but maybe you can define a constant for " " at the beginning of the component, and set the variable within the { } brackets instead.